inc/sainexthopgroup.h

```diff
@@ -74,6 +80,69 @@ typedef enum _sai_next_hop_group_member_observed_role_t
 
 } sai_next_hop_group_member_observed_role_t;
 
+/**
+ * @brief Attribute data for #SAI_NEXT_HOP_GROUP_MEMBER_ATTR_ADMIN_STATE
+ */
+typedef enum _sai_next_hop_group_member_admin_state_t
+{
+    /** Active */
+    SAI_NEXT_HOP_GROUP_MEMBER_ADMIN_STATE_ACTIVE,
+
+    /** Inactive. Graceful shutdown */
+    SAI_NEXT_HOP_GROUP_MEMBER_ADMIN_STATE_INACTIVE,
+
+    /** Force inactive. Disruptive to the connections */
+    SAI_NEXT_HOP_GROUP_MEMBER_ADMIN_STATE_FORCE_INACTIVE,
+
+} sai_next_hop_group_member_admin_state_t;
+
+/**
+ * @brief Attribute data for #SAI_NEXT_HOP_GROUP_MEMBER_ATTR_OPER_STATE
+ */
+typedef enum _sai_next_hop_group_member_oper_state_t
+{
+    /**
+     * @brief Inactive. Does not serve any connections and is safe to remove
+     */
+    SAI_NEXT_HOP_GROUP_MEMBER_OPER_STATE_INACTIVE,
```

Comment:
it is not clear the criteria to determine if a member is inactive or not. how to you decide if a session is active or not, where to define the timeout period? Do you do the tcp connection state tracking?